enscalator.gemspec

```diff
@@ -35,16 +35,17 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 3.2.0'
   spec.add_development_dependency 'rspec-expectations', '~> 3.2.0'
   spec.add_development_dependency 'rspec-mocks', '~> 3.2.1'
-  spec.add_development_dependency 'rubocop', '~> 0.47.1'
-  spec.add_development_dependency 'rubocop-rspec', '~> 1.10.0'
+  spec.add_development_dependency 'rubocop', '~> 0.52.0'
+  spec.add_development_dependency 'rubocop-rspec', '~> 1.21.0'
   spec.add_development_dependency 'coveralls', '~> 0.8.19'
 
   spec.add_runtime_dependency 'aws-sdk', '2.1.36'
-  spec.add_runtime_dependency 'cloudformation-ruby-dsl', '~> 1.0'
-  spec.add_runtime_dependency 'nokogiri', '~> 1.6.6.2'
-  spec.add_runtime_dependency 'trollop', '~> 2.1'
-  spec.add_runtime_dependency 'activesupport', '~> 4.2'
-  spec.add_runtime_dependency 'ipaddress', '~> 0.8'
-  spec.add_runtime_dependency 'semantic', '~> 1.4.0'
-  spec.add_runtime_dependency 'ruby-progressbar', '~> 1.7.5'
+  spec.add_runtime_dependency 'cloudformation-ruby-dsl', '1.2.1'
```

Comment:
Isn't that being too specific?
~> 1.2 should be sufficient.

Reply:
I'll adjust some of them to be more generic, but not all.